lib/led/ledcontrol.js

```diff
@@ -772,14 +772,9 @@ Controllers = {
 
   DEFAULT: {
     OP: {},
-    initialize: function(opts) {
+    initialize: function() {
 
-      this.pins = {
-        data: opts.pins.data,
-        clock: opts.pins.clock,
-        cs: opts.pins.cs || opts.pins.latch
-      };
```

Comment:
For anyone wondering why this is removed, @HipsterBrown and I discovered that it's unnecessarily duplicating the efforts of `Board.Component`